src/dialog/marisa.c

```diff
@@ -127,10 +357,102 @@ DIALOG_TASK(marisa, Stage5PreBoss) {
 
 	ACTOR_LEFT(marisa);
 	ACTOR_RIGHT(iku);
+	HIDE(iku);
+
+	//FACE(marisa, inquisitive);
+	FACE(marisa, surprised);
+	MSG(marisa, "This place is fascinatin’! Is this space-time magic?");
+	MSG(marisa, "What kinda energy source would they even use for this?");
+	MSG(marisa, "I gotta tell Patchy!");
 
 	EVENT(boss_appears);
+	MSG_UNSKIPPABLE(iku, 180, "I would not recommend getting too enamoured, witch.");
+	SHOW(iku);
+	MSG(iku, "Regardless, I’m surprised to see you here. How have you been?");
+
+	MSG(marisa, "What textbook do ya figure this’d be in…?");
+	MSG(marisa, "Hmm… probably a major illusion of some kind…");
+	MSG(marisa, "That’d be easier than making the bricks weightless, and it’s a good disguise…");
+
+	TITLE(iku, "Nagae Iku", "Fulminologist of the Heavens");
+	FACE(iku, smile);
+	MSG(iku, "U-um, pardon me. Are you conversing with yourself?");
+
+	FACE(marisa, normal);
+	MSG(marisa, "Huh? No, I was askin’ ya a question.");
+
+	MSG(iku, "Would that be your theory about this tower, perhaps?");
+
+	MSG(marisa, "It’d make sense though, right?");
+	//FACE(marisa, inquisitive);
+	MSG(marisa, "It appeared outta nowhere. If it were real brick, the air displacement alone woulda caused a huge noise…");
+	MSG(marisa, "And it’d take so much power, too.");
+	FACE(marisa, smug);
+	MSG(marisa, "Definitely a major illusion.");
+
+	MSG(iku, "I feel as though you are underestimating her.");
+	FACE(iku, eyes_closed);
+	MSG(iku, "This tower is affecting people mentally, after all.");
+	MSG(iku, "The technology on display here is beyond our current comprehension.");
+
+	FACE(marisa, surprised);
+	MSG(marisa, "Whoa, so ’yer sayin’ they have enough power for all that? That this is REAL brick?!");
```

Comment:
Hm. I might be making shit up here, but I remember in an earlier revision this line went something like just:
> "Whoa, so ’yer sayin’ that this is REAL brick?!"

And I thought it was funnier that way.

Reply:
Agreed, and changed!